server/src/integration/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationLampIT.java

```diff
@@ -31,7 +33,10 @@ protected void onSuccess(File outputDir) {
 
     @Override
     protected void onFailure(File outputDir, Exception e) {
-        fail();
+        //if the cause for transformation failure is amazonEC2Exception its fine because of the credentials 
+        if (!(e instanceof TransformationFailureException && e.getCause() instanceof AmazonEC2Exception)) {
```

Comment:
Please consider using assumptions to prevnt a test from beeing executed if the credentials are not set.

you can use the check assumptions method in this test to do this.